### ISSUE TITLE
Exception when using reverse_multiple=False

### ISSUE THREAD

**Issue by kmierzeje** [AlexHill/django-relativity#13]
I have this model:
```
sep=Value('/')
class TreeNode(Model):
    name = CharField(max_length=30)
    parent_id = CharField(max_length=512, db_index=True, blank=True, default='')    
    
    children = Relationship('self',
                            predicate=Q(parent_id=Concat(L('parent_id'), sep, L('name'))),
                            reverse_multiple=False,
                            related_name='parent')
```

When trying to access parent field:
```
obj=TreeNode.objects.last()
print(obj.parent)
```
I get this stack trace:
```
Traceback (most recent call last):
  File "C:\apps\Python37\lib\site-packages\django\db\models\fields\related_descriptors.py", line 401, in __get__
    rel_obj = self.related.get_cached_value(instance)
  File "C:\apps\Python37\lib\site-packages\django\db\models\fields\mixins.py", line 13, in get_cached_value
    return instance._state.fields_cache[cache_name]
KeyError: 'parent'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\apps\Python37\lib\site-packages\django_admin_shell\views.py", line 42, in run_code
    exec(code)
  File "<string>", line 11, in <module>
  File "C:\apps\Python37\lib\site-packages\django\db\models\fields\related_descriptors.py", line 407, in __get__
    filter_args = self.related.field.get_forward_related_filter(instance)
  File "C:\apps\Python37\lib\site-packages\relativity\fields.py", line 347, in get_forward_related_filter
    q = self.field.predicate
AttributeError: 'CustomForeignObjectRel' object has no attribute 'predicate'
```

**Comment by kmierzeje** [comment]
I managed to fix it by changing `related_accessor_class` in `Relationship` to something like this:

```
class ReverseManyToOneRelationshipDescriptor(ReverseManyToOneDescriptor):
    def __get__(self, instance, cls=None):
        if instance is None:
            return self

        return self.related_manager_cls(instance).get()
    
    def _get_set_deprecation_msg_params(self):
        return (
            'reverse side of a related set',
            'related fields',
        )
```

**Comment by AlexHill** [comment]
Thanks for the bug report! I've merged a fix and will release 0.2.2 in a minute.

**Comment by AlexHill** [comment]
0.2.2 now released to PyPI with a fix for this issue. Thanks again!
